nixos/modules/misc/ids.nix

```diff
@@ -276,6 +276,7 @@
       telegraf = 256;
       gitlab-runner = 257;
       postgrey = 258;
+      hound = 259;
```

Comment:
What sort of data does `hound` generate? Is it not sufficient to ensure eventually consistent ownership via `preStart` or similar mechanisms?